app/views/devise/registrations/new.html.erb

```diff
@@ -28,64 +28,110 @@
                                 <div class="row justify-content-center row">
                                     <div class="col-md-10 col-lg-6 col-xl-5 order-2 order-lg-1 col">
                                         <p class="text-center h1 fw-bold mb-5 mx-1 mx-md-4 mt-4">Sign up</p>
-                                        <form class="form mx-1 mx-md-4">
+                                        <%# <form class="form mx-1 mx-md-4">
```

Comment:
What is happening here? Is this `%#` doing something for you?

Reply:
I'm not quite ready to delete those comments yet. I just needed to get the working login page before our mentor session today